README.md

```diff
@@ -16,7 +16,7 @@ Documentation and downloads: https://ensmallen.org
 ### Requirements
 
 * C++ compiler with C++14 support
-* Armadillo: https://arma.sourceforge.net
+* Armadillo (https://arma.sourceforge.net) >= 10.8.2
```

Comment:
```suggestion
* Armadillo (https://arma.sourceforge.net), version 10.8.2 or later
```

The hanging '>= 10.8.2' rubs me the wrong way, somehow.

Reply:
Works for me :+1: